sphinx/screens/dashboard/dashboard/src/main/java/chat/sphinx/dashboard/ui/DashboardFragment.kt

```diff
@@ -80,6 +81,11 @@ internal class DashboardFragment : MotionLayoutFragment<
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+
+        binding.layoutDashboardChats.swipeContainer.setOnRefreshListener{
+            viewModel.networkRefresh()
+            binding.layoutDashboardChats.swipeContainer.setRefreshing(false);
+        }
```

Comment:
- Move to `setupChats()` method
- Change to new id and consolidate using apply
- Use property access syntax instead of setter method (isRefreshing = false)
```suggestion
        binding.layoutDashboardChats.layoutSwipeRefreshChats.apply {
            setOnRefreshListener {
                viewModel.networkRefresh()
                isRefreshing = false
            }
        }
```